sylt-compiler/src/typechecker.rs

```diff
@@ -655,7 +654,12 @@ impl TypeChecker {
             | StatementKind::StatementExpression { .. }
             | StatementKind::Unreachable
             | StatementKind::EmptyStatement => {
-                panic!("Illegal outer statement! Parser should have caught this")
+                panic!(
```

Comment:
```suggestion
                unreachable!(
```
This is supposed to unreachable right?

Reply:
yes :+1: